integration/commands/dev.go

```diff
@@ -87,25 +87,24 @@ func RunOktetoUp(oktetoPath string, upOptions *UpOptions) (*UpCommandProcessResu
 	}, nil
 }
 
-func RunOktetoUpAndWait(oktetoPath string, upOptions *UpOptions) error {
-
+func RunOktetoUpAndWait(oktetoPath string, upOptions *UpOptions) (bytes.Buffer, error) {
```

Comment:
```suggestion
func RunOktetoUpAndWaitWithOutput(oktetoPath string, upOptions *UpOptions) (bytes.Buffer, error) {
```

could we rename the function if now is returning the output? or perhaps create a similar one with output and keep the original if is used on other places?

Reply:
@teresaromero this function is only used for this test, would you still go for the `WithOutput` suffix in that case?

Reply:
"only for test" but i think we should be consistent. We dont have a guidelines for naming yet but i think is better if we change it like that. No hard feelings

Reply:
@teresaromero "only for _this_ test" was a reply to your feedback: "or perhaps create a similar one with output and keep the original if is used on other places" - my point is, no need to create a new one because it's only used once

Reply:
I agree on the name, it's more explicit 👍 https://github.com/okteto/okteto/pull/3764/commits/512e05d846698856ab86dd1978732d9216e61c6e